package.json

```diff
@@ -1,18 +1,24 @@
 {
-    "name": "@ng-web-apis/common",
-    "version": "2.1.0",
+    "name": "ng-web-apis",
+    "private": true,
     "scripts": {
-        "postinstall": "husky install",
-        "build": "ng build",
+        "postinstall": "husky install && ngcc --async",
+        "build": "nx build",
         "postbuild": "node scripts/postbuild.js",
-        "test": "ng test",
-        "lint": "eslint . --fix",
-        "typecheck": "tsc --noEmit --skipLibCheck",
         "release": "standard-version",
         "release:patch": "npm run release -- --release-as patch",
         "release:minor": "npm run release -- --release-as minor",
         "release:major": "npm run release -- --release-as major",
-        "publish": "npm run build && npm publish ./dist/common"
+        "publish": "npm run build && npm publish ./dist/common",
```

Comment:
Should this command to be part of `projects/common/project.json` ?

```
nx publish common
```

Reply:
No, in the future we can use `npx lerna version --conventional-commits --conventional-graduate --yes`